src/ui/components/ServerInfoContent.tsx

```
@@ -345,7 +345,7 @@ const ServerInfoContent = ({
                       )}
                     </Box>
                   )}
-                  <Box fontSize='x10' color='annotation' style={textWrapStyle}>
+                  <Box fontSize='x10' color='font-annotation' style={textWrapStyle}>
```

ESLint

[error] Replace `·fontSize='x10'·color='font-annotation'·style={textWrapStyle}` with `⏎····················fontSize='x10'⏎····················color='font-annotation'⏎····················style={textWrapStyle}⏎··················`

(prettier/prettier)